src/components/MarkdownProvider/components/Configuration.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { UnorderedList, Span, MD } from '@zendeskgarden/react-typography';
+import { Anchor } from '@zendeskgarden/react-buttons';
+import { getColor } from '@zendeskgarden/react-theming';
+import { ComponentDoc } from 'react-docgen-typescript';
+
+interface IPackage {
+  version: string;
+  name: string;
+  packageName: string;
+}
+
+const StyledUnorderedList = styled(UnorderedList)`
+  margin: 0 0 ${p => p.theme.space.xxl};
+  border-bottom: ${p => `${p.theme.borders.sm} ${getColor('grey', 200, p.theme)}`};
+`;
+
+const StyledListItem = styled(UnorderedList.Item)`
+  list-style: none;
+
+  & > div {
+    display: flex;
+    margin: ${p => p.theme.space.base * 2.5}px 0;
+    border-top: ${p => `${p.theme.borders.sm} ${getColor('grey', 200, p.theme)}`};
+    padding: ${p => p.theme.space.base * 2.5}px 0 0;
+  }
+`;
```

Comment:
Prefer using `UnorderedList type="none"` which probably means this component can go away and just provide a

```
const StyledListItemContent = styled.div`
  display: flex;
  ...
`
```

for the child content.

Reply:
Actually, if that suggestion :point_up: is a type error, you also add `list-style-type: none;` to `StyledUnorderedList`. Either way, you should be able to use `UnorderedList.Item` without wrapping.

Reply:
Moved the list style to the parent. To avoid div soup, kept the child div targeting.

Reply:
Thanks for the child `div` reminder, @rossmoody. Sounds good. Btw, that child div exists so that we can re-leverage baseline typography components (`SM`, `MD`, `LG`, etc) within the context of the list.